http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -216,7 +217,11 @@ class ProxyService(
               IO.fromFuture(IO(handleHttpRequest(targetHost, request)))
           }
           r <- if (response.status.isFailure())
-            IO(logger.info(s"Error response for proxied request ${request.uri}: ${response.status}")).as(response)
+            IO(
+              logger.info(
+                s"Error response for proxied request ${request.uri}: ${response.status}, ${Unmarshal(response.entity).to[String]}"
```

Comment:
logging error response body when we get proxy errors

Reply:
Could the response body get large? E.g. I hope the entire notebook content wouldn't be returned in the response body

Reply:
so this is only logged when there's error...so I don't think we'll get a huge response body? but I can also limit the body size just to be safe